core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -281,39 +281,58 @@ public TranslationStatus getTranslationJobStatus(String strTranslationJobID) thr
         try {
           int imported = 0;
           int translated = 0;
+          int approved = 0;
+          int rejected = 0;
           SourceFile[] files = liltApiClient.getFiles(strTranslationJobID);
-          // loop backwards through the list since the most recent files will be at the end.
           for (SourceFile file : files) {
             if (file.labels.contains("status=IMPORTED")) {
               imported++;
             }
             if (file.labels.contains("status=TRANSLATED")) {
               translated++;
             }
+            if (file.labels.contains("approval=APPROVED")) {
+              approved++;
+            }
+            if (file.labels.contains("approval=REJECTED")) {
+              rejected++;
+            }
           }
           boolean hasImported = imported > 0;
           boolean hasTranslated = translated > 0;
+          boolean hasRejected = rejected > 0;
+          if (hasImported && hasTranslated && approved >= imported) {
+            log.warn("lilt: job status for {} is APPROVED", strTranslationJobID);
+            return TranslationStatus.APPROVED;
+          }
+          if (hasImported && hasTranslated && hasRejected) {
+            log.info("lilt: job status for {} is TRANSLATION_IN_PROGRESS", strTranslationJobID);
```

Comment:
Did you mean to `warn` here instead of `info`?
You should probably log `REJECTED` status instead of `TRANSLATION_IN_PROGRESS` and return that too.